src/actions/MessageCenterActions.ts

```diff
@@ -23,56 +23,27 @@ const toNumberArray = (n: numberOrNumberArray) => (Array.isArray(n) ? n : [n]);
 export const MessageCenterActions = {
   addMessage: createAction(
     MessageCenterActionKeys.ADD_MESSAGE,
-    (content: string, groupId: string = DEFAULT_GROUP_ID, messageType: MessageType = DEFAULT_MESSAGE_TYPE) => ({
-      type: MessageCenterActionKeys.ADD_MESSAGE,
-      content,
-      groupId,
-      messageType
-    })
+    resolve => (content: string, groupId: string = DEFAULT_GROUP_ID, messageType: MessageType = DEFAULT_MESSAGE_TYPE) =>
```

Comment:
Read a little more here: https://github.com/piotrwitek/typesafe-actions#3-most-flexible-with-all-helpers
What I understand is that is "helper" that builds the final action object, so that we no longer need to code the `Action` structure and focus on the payload.